src/planner/binder/tableref/bind_table_function.cpp

```diff
@@ -86,7 +86,9 @@ bool Binder::BindTableFunctionParameters(TableFunctionCatalogEntry &table_functi
 			auto node = binder->BindNode(*se.subquery->node);
 			subquery = make_unique<BoundSubqueryRef>(std::move(binder), std::move(node));
 			seen_subquery = true;
-			arguments.emplace_back(LogicalTypeId::TABLE);
+			arguments.emplace_back(LogicalType::TABLE);
+			// add a toxic dummy value so the offsets and lengths of arguments and parameters match
+			parameters.emplace_back(Value(LogicalType::INVALID));
```

Comment:
Because of this added parameter, this will fail because index 0 is no longer the pointer parameter
```
frame #3: 0x000000013497e45c duckdb.cpython-311-darwin.so`duckdb::MapFunction::MapFunctionBind(context=0x00000001040c29f8, input=0x000000016fdf9560, return_types=size=0, names=size=0) at map.cpp:52:46
   49  
   50           auto data_uptr = make_unique<MapFunctionData>();
   51           auto &data = *data_uptr;
-> 52           data.function = (PyObject *)input.inputs[0].GetPointer();
```